app/views/editor/questionRich.html.erb

```diff
@@ -5,21 +5,15 @@
 <link href="/lib/publiclab-editor/dist/PublicLab.Editor.css" rel="stylesheet">
 
 <!-- required for MapModule -->
-<link href="/lib/leaflet/dist/leaflet.css" rel="stylesheet">
-<script src="/lib/leaflet/dist/leaflet.js"></script>
```

Comment:
I think we should keep this and remove from `application` level, because as you are also working on performance improvement, having a light entry page should take less time, right?
When these libraries are required they should be loaded only then (lazy loading), what do you think?
😄 🎉

Reply:
Hi @Tlazypanda any comment on these last questions from @sagarpreet-chadha ? My apologies for replying late to these. Thank you!!!

Reply:
Hey @jywarren @sagarpreet-chadha I totally missed this :sweat_smile: my apologies ...ummm can we do this in a follow up pr because I think the leaflet.js and css files are actually required on many routes since the map is loaded on dashboard/user profile/all editor pages/map routes etc so might have to investigate it more to make sure that removing it won't break anything. What do you think? Thanks :v:

Reply:
That's totally fine. Always game to split up PRs! Does that make this PR otherwise good to merge? Thank you!